app.js

```diff
@@ -34,6 +34,7 @@ app.use(helmet({
 }));
 app.use(bodyParser.json());
 app.use('/client', express.static(path.join(__dirname, 'dist')));
+app.get('/Coral.js', (req, res) => res.sendFile(path.join(__dirname, 'client/coral-embed-api/index.js')));
```

Comment:
Ideally (eventually) this should live in `/dist/` as the ultimate goal is to be able to host these assets on a cdn. This is not possible now, however. 

My thought is to add a simple copy operation to the webpack file to move this static file from it's home in `/client/` to the root of the '/dist/` folder so it can be served via our static file route.